c7n/filters/core.py

```diff
@@ -431,8 +431,7 @@ def process_value_type(self, sentinel, value):
                 # EMR not having more functionality.
                 try:
                     value = parse(value, default=datetime.now(tz=tzutc()))
-
-                except (AttributeError, TypeError):
+                except:
```

Comment:
note, certain tag values can cause this to barf with a ValueError.

Since we're not catching all exception, those errors bubble up and can interrupt certain policies. It's better if we just catch all exceptions.